### ISSUE TITLE
addPeakMatrix() HDF5 accessibility errors

### ISSUE THREAD

**Issue by AnjaliC4** [GreenleafLab/ArchR#1335]
Hello, I know this has been reported with other functions but I am unable to proceed without a solution. I am working on a cluster. I have tried several things such as - setting threads=1, h5disableFileLocking(), giving more memory, changing the directory paths in R to ArchRproject. Nothing seems to be working. Sometimes I also get this error for specific iterations of addPeakMatrix. 

Error in H5Fopen(file, flags = flags, fapl = fapl, native = native) :
  HDF5. File accessibility. Unable to open file.
Calls: addPeakMatrix ... .sampleName -> h5read -> h5checktypeOrOpenLoc -> H5Fopen
Execution halted

In addPeakMatrix source code - I see the code - not sure if this is causing the issue.
o <- h5closeAll()

Please let me know what's a possible solution- if anyone has faced it.

**Comment by rcorces** [comment]
Hi @AnjaliC4! Thanks for using ArchR! Please make sure that your post belongs in the Issues section. Only bugs and error reports belong in the Issues section. Usage questions and feature requests should be posted in the [Discussions](https://github.com/GreenleafLab/ArchR/discussions) section, not in Issues.<br />
__Before we help you, you must respond to the following questions__ unless your original post already contained this information:
__1.__ If you've encountered an error, have you already searched previous Issues to make sure that this hasn't already been solved?
__2.__ Can you recapitulate your error using the tutorial code and dataset? If so, provide a reproducible example.
__3.__ Did you post your log file? If not, add it now.

**Comment by rcorces** [comment]
try https://github.com/GreenleafLab/ArchR/issues/1183#issuecomment-980781270

**Comment by AnjaliC4** [comment]
Hi, I found that one of my ArrowFiles was empty - I fixed that and then I ran:

h5disableFileLocking()
proj <- addPeakMatrix(proj,force=TRUE,threads=8)
#h5enableFileLocking()
h5closeAll()
se <- getMatrixFromProject(proj2,"PeakMatrix",threads=4)

This worked for me - for anyone who may encounter this.
